doc/source/myrocks/zenfs.rst

```diff
@@ -0,0 +1,80 @@
+.. _zenfs:
+
+=========================================
+RocksDB with ZenFS
+=========================================
+
+Implemented in Percona Server for MySQL 8.0.26-16.
+
+`ZenFS <https://zonedstorage.io/projects/zenfs/>`__ is a file system plugin which places files into zones on a raw zoned block device (ZBD) using the RocksDB File System interface. ZenFS depends on ``libzbd`` and requires a Linux kernel implementation which supports MVMe Zoned namespaces. The kernel must be configured with `zone block device support enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__. 
+
+The following steps successfully run Percona Server with MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``:
+
+.. note::
+
+    The ``<block_device_name>`` can have a short name desigination which is the ``<short_block_device_name>``. For example, the ``block_device_name`` is ``/dev/nvme1n2`` and the short name is ``nvme1n2``.
+
+#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``:
+
+    .. sourcecode:: bash
+    
+     lsblk
+
+#. Create an auxiliary directory for the ZenFS metadata locks, for example, ``/var/lib/mysql_aux``. The directory must be write accessible for the `mysql:mysql` UNIX system user account. Each ZBD must have an individual auxiliary directory.
+
+
+#. Run ``/zenfs mkfs --zbd <short_block_device_name> --aux_path=<path_to_aux_zenfs_dir>`` in the context of the mysql:mysql account to initialize ZenFS on the ZBD. 
+
+    .. sourcecode:: bash
+
+        zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux 
+
+    .. note:: 
+
+        This step initializes the metadata on the block device, ``<block_device_name>``, and creates the ``<path_to_aux_zenfs_dir>`` owned by the `mysql:mysql` account. For example, the ``<block_device_name>`` could be ``/dev/nvme1n2``. The ``<path_to_aux_zenfs_dir>`` is a unique, dedicated directory which allows read and write access to `mysql:mysql`. 
+
+        If you are re-initializing an already initialized ZBD, you must manually clear the ``/var/lib/mysql_aux`` directory and add the additional ``--force`` parameter.
+
+        .. sourcecode:: bash
+
+            zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux  --force
+
+
+
+#. Start Percona Server for MySQL with 
```

Comment:
How do I pass this option? Can we point to some further docs on how to provide such supplemental options for starting the server?

Reply:
Any known way of passing options to `mysqld` should work - probably, putting this option into `my.cnf` is the most straightforward one.

Reply:
Updated.